documentation/docs/changelog.md

```diff
@@ -4,7 +4,7 @@ sidebar_label: Changelog
 slug: changelog.html
 ---
 
-## 5.4.0
+## 5.4.0 (Unreleased)
```

Comment:
The 5.4 docs are only available by selecting that WIP-version using the version selector. All documentation for that version has a notice like this:

<img width="1064" alt="image" src="https://user-images.githubusercontent.com/316857/174537159-1d14a79f-28b0-44f3-941d-4153b05e069e.png">

So I would argue that adding _Unreleased_ is a bit redundant, although it probably doesn't hurt either, as long as we remember to remove it when doing the release 🙂